authentication/authentication-styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 100px;
+    margin-bottom: 50px;
+}
+
+.card {
+    max-width: 50vw;
```

Comment:
- We should not set the `max-width` for the card. The default one is 100% = width of the screen and we should leave it be.